tests/test_randomized.py

```diff
@@ -188,37 +188,37 @@ def test_gl(groups):
             assert np.linalg.norm(grad_map) < 1e-6
 
 
-def test_vrtos_ogl():
```

Comment:
why removing the test? This is testing an overlapping group lasso model by adding to proximal functions (for solvers that allow this)